### ISSUE TITLE
Can't build Android: jcenter - Read timed out

### ISSUE THREAD

**Issue by retyui** [iamolegga/react-native-launch-arguments#48]
- "react-native-launch-arguments": "^3.1.0",

```
Build file 'node_modules/react-native-launch-arguments/android/build.gradle' line: 112

* What went wrong:
A problem occurred configuring project ':react-native-launch-arguments'.
> Could not resolve all files for configuration ':react-native-launch-arguments:implementation'.
   > Could not resolve com.facebook.react:react-native:+.
     Required by:
         project :react-native-launch-arguments
      > Failed to list versions for com.facebook.react:react-native.
         > Unable to load Maven meta-data from https://jcenter.bintray.com/com/facebook/react/react-native/maven-metadata.xml.
            > Could not HEAD 'https://jcenter.bintray.com/com/facebook/react/react-native/maven-metadata.xml'.
               > Read timed out

```

**Comment by RohovDmytro** [comment]
Confirming. The issue appeared for me two days ago.

**Comment by mirocosic** [comment]
+1

**Comment by retyui** [comment]
@RohovDmytro jcenter should work read-only, but incident https://status.gradle.com/incidents/9740r6bzx438

**Comment by GamesOfSummer** [comment]
+1

**Comment by iamolegga** [comment]
should be solved in #49 , please check `3.1.1`
